src/js/helpers/awardAmountChartHelper.js

```diff
@@ -0,0 +1,279 @@
+    subsidyColor,
+    faceValueColor,
+    infrastructureOutlayColor,
+    infrastructureObligatedColor,
+    infrastructureCurrentColor,
+    infrastructurePotentialColor,
+    // Offsets per DEV-5242:
+    lineOffsetsBySpendingCategory,
+    defcTypes
+} from 'dataMapping/award/awardAmountsSection';
+
+import { covidColor, covidObligatedColor } from 'dataMapping/covid19/covid19';
+
+export const getfileCInfo = (fileCType) => {
+    let fileCInfo = null;
+    defcTypes.forEach((item) => {
+        if (item.codeType === fileCType) {
+            fileCInfo = item;
+        }
+    });
+
+    return fileCInfo;
+};
+
+export const getAwardTypeText = (awardType, amountType, fileCType) => {
+    const fileCInfo = getfileCInfo(fileCType);
+    const preText = fileCInfo && Object.keys(fileCInfo)?.length > 0 ? fileCInfo.preText : '';
+    return awardType === "idv" ? `Combined ${preText} ${amountType} Amounts` : `${preText} ${amountType} Amount`;
+};
+
+// TODO: Address with continued award chart refactor
+export const getAwardColor = (overallColor, infrastructureColor, fileCColor, fileCType) => {
+    const fileCInfo = getfileCInfo(fileCType);
+
+    if (fileCInfo?.codeType === "infrastructure") {
+        return infrastructureColor;
+    }
+
+    if (fileCInfo?.codeType === "covid") {
+        return fileCColor;
+    }
+
+    return overallColor;
+};
+
+export const getAwardOutlayRawValue = (data, awardType, fileCType) => {
+    const fileCInfo = getfileCInfo(fileCType);
+
+    if (fileCInfo?.codeType === "covid") {
+        return data._fileCOutlay;
+    }
+
+    if (fileCInfo?.codeType === "infrastructure") {
+        return data._fileCOutlayInfrastructure;
+    }
+
+    return awardType === "idv" ? data._combinedOutlay : data._totalOutlay;
+};
+
+export const getAwardOutlayValue = (data, awardType, fileCType) => {
+    const fileCInfo = getfileCInfo(fileCType);
+
+    if (fileCInfo?.codeType === "covid") {
+        return data.fileCOutlayAbbreviated;
+    }
+
+    if (fileCInfo?.codeType === "infrastructure") {
+        return data.infrastructureOutlayAbbreviated;
+    }
+
+    return awardType === 'idv' ? data.combinedOutlayAbbreviated : data.totalOutlayAbbreviated;
+};
+
+export const getAwardObligatedRawValue = (data, awardType, fileCType) => {
+    const fileCInfo = getfileCInfo(fileCType);
+
+    if (fileCInfo?.codeType === "covid") {
+        return data._fileCObligated;
+    }
+
+    if (fileCInfo?.codeType === "infrastructure") {
+        return data._fileCObligatedInfrastructure;
+    }
+
+    return data._totalObligation;
+};
+
+export const getAwardObligatedValue = (data, awardType, fileCType) => {
+    const fileCInfo = getfileCInfo(fileCType);
+
+    if (fileCInfo?.codeType === "covid") {
+        return data.fileCObligatedAbbreviated;
+    }
+
+    if (fileCInfo?.codeType === "infrastructure") {
+        return data.infrastructureObligationAbbreviated;
+    }
+
+    return data.totalObligationAbbreviated;
+};
+
+// New Bar chart
```

Comment:
Are these comments still needed?
Or maybe moved to describe each block in the code below, like line 134?

Reply:
I added detail to the comment.  This is important to point out because I had to clean up a lot of unused props where weren't using the above pattern. This caused a lot of confusion